crates/sc-consensus-subspace/src/lib.rs

```diff
@@ -1109,7 +1104,7 @@ where
             // Wait for all the acknowledgements to finish.
         }
 
-        Ok(import_result)
+        self.inner.import_block(block).await
```

Comment:
Is there some issue with sending the notification after this `self.inner.import_block(block).await` (except the time lag between import completion and sending the notification)? 

I am missing why it has to be sent before the import, which might actually fail afterwards (this being a behavior change if someone depends on the notification to indicate that import has finished successfully)

Reply:
Did you read https://github.com/subspace/subspace/issues/871#issuecomment-1722745138?

The behavior chance is acceptable here, it was never really meant to be fired after block import or else notification would have been "imported", not "importing". And "imported" already exists, the only difference is it doesn't prevent block import from moving forward.

Reply:
In fact in domains both "importing" and "imported" subscriptions are used together to achieve block import backpressure.

Reply:
Thanks, now I understand the importing vs imported distinction. IIUC, the pre-processing that was supposed to happen before the import was happening out of order sometimes